File: Project_1/project1_code/corona_dirk/plot_populations.py
from numpy import *
from pylab import *

classes = ["S","E","I","R","D"]

t = loadtxt('time.txt')

for c in classes :
	y = loadtxt(c + ".txt")
	plot(t,y,label = c)

xlabel(r'$Time$')
ylabel(r'$Individuals$')

grid(True)
legend(loc = "best")

# savefig("../../figs/SEIR_fe_" + str(t.size-1) + ".png")
# savefig("../../figs/SEIR_dirk_" + str(t.size-1) + ".png")

show()

File: Project_1/project1_code/corona_dirk/plot_populations.m
clc %to clear the command
close all % to close eventual previous windows
clear all % to clear the workspace

FolderName = 'build/';

S = load([FolderName,'S.txt']);
E = load([FolderName,'E.txt']);
I = load([FolderName,'I.txt']);
R = load([FolderName,'R.txt']);
D = load([FolderName,'D.txt']);
t = load([FolderName,'time.txt']);

plot(t, S, 'b', t, E, 'r', t, I, 'g', t, R, 'k', t, D, 'm')
xlabel("Time");
ylabel("Individuals");

legend('S', 'E','I', 'R', 'D')